.github/workflows/latest.yaml

```diff
@@ -11,12 +11,24 @@ jobs:
     steps:
     - name: Checkout
       uses: actions/checkout@v2
+
+    - name: Docker Login
+      uses: docker/login-action@v1
+      with:
+        registry: ${{ secrets.DOCKER_REGISTRY }}
+        username: ${{ secrets.DOCKER_REGISTRY_USER }}
+        password: ${{ secrets.DOCKER_REGISTRY_TOKEN }}
+
     - name: Lint
-      uses: actions-contrib/golangci-lint@master
+      uses: golangci/golangci-lint-action@v2
       with:
-        args: run
+        version: v1.32.2
```

Comment:
i'm pretty sure timeout is missing :-)

Reply:
Timeout is only required when there are Kubernetes dependencies. ;)